src/pallene/pallenec.lua

```diff
@@ -34,7 +34,8 @@ do
     )
 
     -- No Pallene tracebacks
-    p:flag("--use-traceback", "Use function traceback for debugging")
+    p:flag("--use-traceback",    "Use function traceback for debugging using Pallene Tracer")
```

Comment:
We should probably use user-friendly messages and omit mentions of implementation details (e.g: use of Pallene tracer).

```suggestion
    p:flag("--use-traceback",    "Show a call-stack trace on error")
```

Reply:
I think it's fine. In Pallene we are just accumulating the traces, which is then shown by `pallene-tracer` script or `xpcall` with `pallene_tracer_debug_traceback`. Mention of Pallene Tracer might spark interest in peoples mind :)